modules/effects/arrow-effects-instances/src/test/kotlin/arrow/effects/IOTest.kt

```diff
@@ -243,7 +243,7 @@ class IOTest : UnitSpec() {
     }
 
     "unsafeRunTimed times out with None result" {
-      val never = IO.async<Int> { }
+      val never = IO.async().never<Unit>().fix()
```

Comment:
*squint* I wrote it, but it's still ugly. Maybe it's being generated now and it's just an import?

Reply:
No idea